docs/conf.py

```diff
@@ -35,8 +35,16 @@
     "sphinx_copybutton",
     "sphinx.ext.doctest",
     "sphinx.ext.intersphinx",
+    "sphinxext.opengraph",
 ]
 
+# Open Graph metadata
+ogp_title = "urllib3 documentation"
+ogp_site_url = "https://urllib3.readthedocs.io"
```

Comment:
For some reason, the Facebook sharing debugger gets 404s for `/latest` URLs:
https://urllib3.readthedocs.io/en/latest/advanced-usage.html

https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Furllib3.readthedocs.io%2Fen%2Flatest%2Fadvanced-usage.html

I wonder if it's this line causing mixup with the canonical link. 

We don't use it for Pillow (see above) and https://github.com/wpilibsuite/sphinxext-opengraph#options says:
> Users hosting documentation on Read The Docs *do not* need to set any of the following unless custom configuration is wanted.